examples/nlp/gpt/train_gpt_ppo_actor.py

```diff
@@ -130,6 +130,7 @@ def main(cfg) -> None:
         gbs=cfg.model.ppo.num_val_samples,
         collate_fn=collate_fn,
         load_gbs=False,
+        use_random_sampler=False,
```

Comment:
i think we should also reflect this change on RM, SFT and DPO

Reply:
Done

Reply:
Done